### ISSUE TITLE
Deployment of a VM with a DATADISK on Ceph sets QCOW2 format

### ISSUE THREAD

**Issue by slavkap** [apache/cloudstack#5531]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Ceph storage
~~~

##### CLOUDSTACK VERSION

~~~
4.15.3.0/main but may exist on previous versions
~~~

##### OS / ENVIRONMENT

CentOS 7
KVM hypervisors

##### SUMMARY

After deploying a VM with a `DATADISK` (both volumes - ROOT/DATADISK  are on Ceph) the `DATADISK` gets `QCOW2` format

```
select id, pool_type, storage_provider_name from storage_pool where id=203;
+-----+-----------+-----------------------+
| id  | pool_type | storage_provider_name |
+-----+-----------+-----------------------+
| 203 | RBD       | DefaultPrimary        |
+-----+-----------+-----------------------+
1 row in set (0.00 sec)

SELECT format, pool_id, name, instance_id FROM cloud.volumes where instance_id=5360;
+--------+---------+-----------+-------------+
| format | pool_id | name      | instance_id |
+--------+---------+-----------+-------------+
| RAW    |     203 | ROOT-5360 |        5360 |
| QCOW2  |     203 | DATA-5360 |        5360 |
+--------+---------+-----------+-------------+
2 rows in set (0.00 sec)
```

##### STEPS TO REPRODUCE

~~~
Deploy a virtual machine with a `datadisk` on Ceph
~~~

##### EXPECTED RESULTS

~~~
The format of the volume should be in `RAW` format
~~~

##### ACTUAL RESULTS

~~~
in the database, the volume's format is `QCOW2`
~~~

**Comment by sureshanaparti** [comment]
Hi @slavkap can you work on the changes for this issue?

**Comment by slavkap** [comment]
Hi @sureshanaparti, if nobody works on a fix, I will try to fix it on time for 4.16.1.0

**Comment by sureshanaparti** [comment]
> Hi @sureshanaparti, if nobody works on a fix, I will try to fix it on time for 4.16.1.0

ok, thank you @slavkap

**Comment by sureshanaparti** [comment]
Fixed in #5801
